kubeadm/startup-cp

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
do we want to use the same flags in all bash scripts?
```
set -o errexit
set -o nounset
set -o pipefail
set -o xtrace
```

Reply:
these are concatenated over the setup.sh so they inherit it's settings.